teleport/datadog_checks/teleport/check.py

```diff
@@ -4,6 +4,42 @@
 
 from datadog_checks.base import OpenMetricsBaseCheckV2
 
+PROXY_METRICS_MAP = {
```

Comment:
Do you think we could move all these maps to a different file (e.g. metrics.py)?